router_test.go

```
@@ -396,6 +399,140 @@ func Test_App_Rebuild_Tree(t *testing.T) {
+
+	resp, err = app.Test(httptest.NewRequest(MethodGet, "/dynamically-defined", nil))
+	require.NoError(t, err, "app.Test(req)")
+	require.Equal(t, http.StatusOK, resp.StatusCode, "Status code")
+	require.Equal(t, uint32(3), app.handlersCount)
+	require.Equal(t, uint32(2), app.routesCount)
+
+	verifyRouteHandlerCounts(app, t)
+}
+
+// this is taken from app.printRoutesMessage()
+func verifyRouteHandlerCounts(app *App, t testing.TB) {
```

golangci-lint (1.62.2)

parameter testing.TB should be the first or after context.Context

(thelper)